### DISCUSSION TITLE
another engine new to me

### DISCUSSION THREAD

**Discussion by zephyr-zeph** [YoshiCrafter29/YoshiCrafterEngine#127] [General]
might try it out

**Comment by itsDARK768** [comment]
I highly suggest this engine for beginners.

**Reply by zephyr-zeph** [reply]
i mostly use psych engine but i gonna try it

**Reply by ghost** [reply]
i recommend captain kirbs tutorials and reading the documentation, always very good.  captain kirbs also been working on a mega tutorial too but it hasn't come out yet, maybe next week once he gets his new pc.
